package.json

```diff
@@ -22,7 +23,6 @@
 		"eslint-config-prettier": "^8.5.0",
 		"eslint-config-react-app": "^7.0.1",
 		"eslint-plugin-jsx-a11y": "^6.6.0",
-		"husky": "^8.0.1",
```

Comment:
I removed this from my end and tried reinstalling it because something broke in my Git setup and I was getting errors. I've fixed this now and Husky is back in.